FleischBot.py

```diff
@@ -4,27 +4,50 @@
 with open ("Fleisch.json", "r+") as fd:
     Kontos=json.load(fd)
 
+
 FleischMessages = [
-	"Und schon wieder ein totes Tier...",
-	"Der Meeresspiegel steigt! 🌊",
-	"Iss auch mal einen Salat!",
-	"🐷 REDRUM",
-	"10 years from now... - Remember how we used to have snow in the winter? 😔",
+    "Und schon wieder ein totes Tier...",
+    "Der Meeresspiegel steigt! 🌊",
+    "Iss auch mal einen Salat!",
+    "🐷 REDRUM",
+    "10 years from now... - Remember how we used to have snow in the winter? 😔",
     "20 years from now... - Remember how we were able to drink water from the sink? 💧",
     "50 years from now... - grandpa/ma, what's a snowman? ☃️",
     "100 years fro-; OH, never mind, you died :-) 🌄"
 ]
 
+
 def Fleisch(bot,update):
     id=update.message.from_user.id
     id=str(id)
-    update.message.reply_text(random.choice(FleischMessages))
-    if id in Kontos:
-        Kontos[id] = Kontos[id]+1
+    commands = update.message.text.split(" ")[1:]
+    if len(commands) > 1:
+        return update.message.reply_text("Don't give so many args!")
+    try:
+        amount = int(commands[0])
+    except ValueError:
+        return update.message.reply_text(
+        	"Specify a valid integer for the amount!"
+        )
+    except IndexError:
+        update.message.reply_text(random.choice(FleischMessages))
+        if id in Kontos:
+            Kontos[id] = Kontos[id]+1
+        else: 
+            Kontos[id] = 1
+    else:
+        update.message.reply_text(random.choice(FleischMessages))
+        if id not in Kontos:
+            Kontos[id] = 1
+        else:
+            if id + "-amount" not in Kontos.keys():
+                Kontos[id + "-amount"] = list()
+            Kontos[id + "-amount"].append(amount)
```

Comment:
The meat consumption history list is generated but never used. Why even bother tracking it?